testdata/serviceconfig.yml

```diff
@@ -21,7 +21,7 @@ rules:
           method: api.method | "unknown"
           response_code: response.http.code | "unknown"
       - descriptor_name:  request_latency
-        value: response.latency | "unknown"
+        value: response.latency | 0
```

Comment:
I think zero is better because we don't do any handling of this value ourselves, we just pass it downstream systems (how does promtheus handle a NaN? I suspect not well). We're slated to come back and look at how we handle missing attributes as a whole, so I see this as a temporary fix.